web-server-lib/web-server/private/timer.rkt

```diff
@@ -18,73 +18,84 @@
 ;; The application may further adjust its allotted timeout for handling
 ;; requests by using the dispatcher in the dispatch-timeout module.
 
-(struct timer-manager (thread timer-ch))
-(define-struct timer (tm evt expire-seconds action)
-  #:mutable)
+(define-logger web-server/timer)
+
+(struct timer-manager (thd))
```

Comment:
Is this struct necessary now?

Reply:
No, but I kept it to minimize the changes since it probably doesn't add much overhead (relative to other things). I'll remove it if necessary when I do some more profiling.